shell/platform/windows/win32_flutter_window.cc

```diff
@@ -147,6 +151,22 @@ void Win32FlutterWindow::OnClose() {
   messageloop_running_ = false;
 }
 
+void Win32FlutterWindow::OnFontChange() {
```

Comment:
@kpozin  Here is the current design for system font api.
When system font changes:
1. Use FlutterEngineReloadSystemFonts in embedder.h to reload system font and clear caches
2. send system channel message {"type": "fontsChange"} to "flutter/system"  flutter framework.

Noted that you will probably need to implement fuchsia version of loading default fonts.
see:   third_party/txt/src/txt/platform_windows.cc
FlutterEngineReloadSystemFonts relys on this function to retrieve the latest system fonts. If there is any caching happen under this layer, the FlutterEngineReloadSystemFonts will still load the old value.

Let me know if this looks ok to you.

Reply:
Yes, this seems fine to me.

Reply:
Thanks for confirming, I will let you know if anything change.

Reply:
@kpozin a little update
you will only need to call this embedder api https://github.com/flutter/engine/blob/aadd5a346bb5fcf970a90e3c98d55818b4734e48/shell/platform/embedder/embedder.h#L1211
I make it so it will send the system message as part of the api.
However, you will still need to implement fuchsia version of loading default fonts.
https://github.com/flutter/engine/blob/master/third_party/txt/src/txt/platform_fuchsia.cc